[2023-11-21T00:39:09.140+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21T00:39:09.150+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21T00:39:09.151+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T00:39:09.163+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-20 00:00:00+00:00
[2023-11-21T00:39:09.170+0000] {standard_task_runner.py:57} INFO - Started process 119 to run task
[2023-11-21T00:39:09.177+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpd30kqn3u']
[2023-11-21T00:39:09.182+0000] {standard_task_runner.py:85} INFO - Job 126: Subtask load_data
[2023-11-21T00:39:09.262+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [running]> on host c6d84631e1b9
[2023-11-21T00:39:09.345+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-20T00:00:00+00:00'
[2023-11-21T00:39:12.637+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T00:39:12.700+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231120T000000, start_date=20231121T003909, end_date=20231121T003912
[2023-11-21T00:39:12.775+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T00:39:12.824+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-21T00:44:28.841+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21T00:44:28.868+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21T00:44:28.869+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T00:44:28.902+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-20 00:00:00+00:00
[2023-11-21T00:44:28.928+0000] {standard_task_runner.py:57} INFO - Started process 190 to run task
[2023-11-21T00:44:28.964+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '171', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpq59k8tgb']
[2023-11-21T00:44:28.975+0000] {standard_task_runner.py:85} INFO - Job 171: Subtask load_data
[2023-11-21T00:44:29.184+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [running]> on host 51d09cf321f5
[2023-11-21T00:44:29.658+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-20T00:00:00+00:00'
[2023-11-21T00:44:33.106+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T00:44:33.131+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231120T000000, start_date=20231121T004428, end_date=20231121T004433
[2023-11-21T00:44:33.208+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T00:44:33.245+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-21T01:00:24.887+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21T01:00:24.895+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21T01:00:24.896+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T01:00:24.916+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-20 00:00:00+00:00
[2023-11-21T01:00:24.926+0000] {standard_task_runner.py:57} INFO - Started process 188 to run task
[2023-11-21T01:00:24.938+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '227', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmphk7hynqo']
[2023-11-21T01:00:24.944+0000] {standard_task_runner.py:85} INFO - Job 227: Subtask load_data
[2023-11-21T01:00:25.049+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-20T00:00:00+00:00 [running]> on host e39dda9a9deb
[2023-11-21T01:00:25.165+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-20T00:00:00+00:00'
[2023-11-21T01:00:29.004+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T01:00:29.057+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231120T000000, start_date=20231121T010024, end_date=20231121T010029
[2023-11-21T01:00:29.147+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T01:00:29.195+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
